.storybook/stories.css

```diff
@@ -18,7 +18,8 @@
   box-shadow: 0px 3px 10px rgba(0, 0, 0, 0.5);
 }
 
-#story-root > .ant-form {
+#story-root > .ant-form,
+#story-root > .ant-table-simple {
```

Comment:
`ant-table-simple` is a lie. It doesn't exist in AntD.
It's a custom version I build for the registry.

You can simply use `.ant-table`